src/toolkits/object_detection/object_detector.hpp

```diff
@@ -39,6 +39,7 @@ class EXPORT object_detector: public ml_model_base {
 
   void train(gl_sframe data, std::string annotations_column_name,
              std::string image_column_name, variant_type validation_data,
+             std::vector<std::string> class_labels,
```

Comment:
As discussed above, let's just include this `options`. Almost everything optional should be in options.